walkthroughs/week-0-setup/github-setup-walkthrough.md

```diff
@@ -71,7 +71,6 @@ To link this local copy to your GitHub project, execute this command:
 ```bash
 git remote add origin YOUR_URL_HERE
 ```
-
```

Comment:
Hey you looks like you removed a line from the walkthrough file by accident. It won't break anything but it's generally good practice to avoid including unrelated changes in a PR. I think it's good to try to get out of the habit of `git add .` or `git commit -a` to add all files without checking them. I generally try to add files/folders individually to make sure I'm getting the specific files I meant to edit.